diff --git a/Users/sunzhiming/Desktop/pb.go b/pb.go
index 97e7b99..55718a2 100644
--- a/Users/sunzhiming/Desktop/pb.go
+++ b/pb.go
@@ -128,6 +128,14 @@ func (pb *ProgressBar) Increment() int {
        return pb.Add(1)
 }

+func (pb *ProgressBar) GetCurrent() int64 {
+       return pb.current
+}
+
+func (pb *ProgressBar) GetFinish() bool {
+       return pb.isFinish
+}
+
 // Set current value
 func (pb *ProgressBar) Set(current int) *ProgressBar {
        return pb.Set64(int64(current))